production/helm/loki/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Entries should be ordered as follows:
 
 Entries should include a reference to the pull request that introduced the change.
 
+[//]: # (<AUTOMATED_UPDATES_LOCATOR> : do not remove this line. This locator is used by the CI pipeline to automatically create a changelog entry for each new Loki release. Add other chart versions and respective changelog entries below this line.)
```

Comment:
Hey @TheRealNoob, is there a reason for this line being duplicated?

Reply:
good spot.  i was fixing a typo across 4 instances of this line.  must have left a duplicate in on this case somehow.